FileInfoLoader.h

```diff
@@ -15,59 +15,58 @@
     FileInfoLoader(const std::string& filename);
-    ~FileInfoLoader() {}
 
-    bool fillFileInfo(CARTA::FileInfo* fileInfo);
-    bool fillFileExtInfo(CARTA::FileInfoExtended* extInfo, std::string& hdu, std::string& message);
+    bool FillFileInfo(CARTA::FileInfo* file_info);
+    bool FillFileExtInfo(CARTA::FileInfoExtended* ext_info, std::string& hdu, std::string& message);
 
 private:
-    casacore::ImageOpener::ImageTypes fileType(const std::string& file);
-    CARTA::FileType convertFileType(int ccImageType);
+    casacore::ImageOpener::ImageTypes FileType(const std::string& file);
+    CARTA::FileType ConvertFileType(int cc_image_type);
 
     // FileInfo
-    bool getHduList(CARTA::FileInfo* fileInfo, const std::string& filename);
+    bool GetHduList(CARTA::FileInfo* file_info, const std::string& filename);
 
     // ExtFileInfo
-    bool fillHdf5ExtFileInfo(CARTA::FileInfoExtended* extInfo, std::string& hdu, std::string& message);
-    bool fillFITSExtFileInfo(CARTA::FileInfoExtended* extInfo, std::string& hdu, std::string& message);
-    bool fillCASAExtFileInfo(CARTA::FileInfoExtended* extInfo, std::string& message);
+    bool FillHdf5ExtFileInfo(CARTA::FileInfoExtended* ext_info, std::string& hdu, std::string& message);
+    bool FillFitsExtFileInfo(CARTA::FileInfoExtended* ext_info, std::string& hdu, std::string& message);
+    bool FillCasaExtFileInfo(CARTA::FileInfoExtended* ext_info, std::string& message);
 
     // Computed entries
-    void addComputedEntries(CARTA::FileInfoExtended* extInfo, const std::string& xyCoords, const std::string& crPixels,
-        const std::string& crCoords, const std::string& crRaDec, const std::string& radeSys, const std::string& specSys,
-        const std::string& bunit, const std::string& axisInc, const std::string& rsBeam);
-    void addShapeEntries(CARTA::FileInfoExtended* extendedInfo, const casacore::IPosition& shape, int chanAxis, int stokesAxis);
-    void findChanStokesAxis(const casacore::IPosition& dataShape, const casacore::String& axisType1, const casacore::String& axisType2,
-        const casacore::String& axisType3, const casacore::String& axisType4, int& chanAxis, int& stokesAxis);
+    void AddComputedEntries(CARTA::FileInfoExtended* ext_info, const std::string& xy_coords, const std::string& cr_pixels,
+        const std::string& cr_coords, const std::string& cr_ra_dec, const std::string& rade_sys, const std::string& spec_sys,
+        const std::string& bunit, const std::string& axis_inc, const std::string& rs_beam);
+    void AddShapeEntries(CARTA::FileInfoExtended* extended_info, const casacore::IPosition& shape, int chan_axis, int stokes_axis);
+    void FindChanStokesAxis(const casacore::IPosition& data_shape, const casacore::String& axis_type_1, const casacore::String& axis_type_2,
+        const casacore::String& axis_type_3, const casacore::String& axis_type_4, int& chan_axis, int& stokes_axis);
 
     // casacore::Record attributes
+    // TODO: what are these for?
```

Comment:
Some HDF5 images have attributes that should be numeric but are strings instead.  These return the values as int or double.

Reply:
`FileInfoLoader::getDoubleAttribute` isn't implemented though, and not used anywhere (instead, it seems like Hdf5Attributes::GetDoubleAttribute` is used.

Reply:
This was moved from FileInfoLoader to Hdf5Attributes but mistakenly not removed from the .h file.  I fixed this and removed the unused _region local variable from CartaHdf5Image.